scss-style-guide.md

```diff
@@ -0,0 +1,100 @@
+# SCSS Style-Guide
+
+We use BEM with unlimited component nesting.
+
+## Glossar
+
+### Block `.button`
+
+### Element`.button__text`
+
+### Modifier`.button--green`
+
+
+
+## Ordering of styles in scss files
+
+The ordering of the elements should be related to the top to bottom occurances of the class in html.
+
+1. `@extend`
+2. `@include`
+3. Elements
+4. Modifiers
+
+### Example
+
+```
+.block {
+	// first extend
+	@extend .box;
+	// second include
+	@include arrow-mixin;
+	// third elements
+	&__child {
+	
+	}
+	// forth modifiers
+	&--blue {
+	
+	}
```

Comment:
Modifiers need to go before children, since they concern the same element. Our linter prevents that only one element or class base because it doesn't understand BEM. However, `button button--blue` would be used on the same element and must go before any children are specified.

Reply:
I do agree, good point.